scout/server/blueprints/cases/views.py

```diff
@@ -95,6 +95,15 @@ def sma(institute_id, case_name):
     return dict(format="html", **data)
 
 
+@cases_bp.route("/<institute_id>/<case_name>/bionano", methods=["GET"])
+@templated("cases/case_bionano.html")
+def bionano(institute_id, case_name):
+    """Visualize case BioNano data - FSHD calls"""
+    institute_obj, case_obj = institute_and_case(store, institute_id, case_name)
+    data = controllers.case(store, institute_obj, case_obj)
```

Comment:
Eh here, like getting all the info required to show a case, invoke a dedicated controller that returns the necessary info?